src/Mail/Mailer.php

```diff
@@ -59,8 +61,14 @@ public function send($view, array $data = [], $callback = null)
             $this->addContent($message, $view, $plain, $raw, $data);
         }
 
+        $messageSubject = $message->getSubject();
+
         call_user_func($callback, $message);
 
+        if ($message->getSubject() === Str::title(Str::snake(class_basename($this), ' ')) {
```

Comment:
Could you add some inline comments explaining what this does and why it's required?